entrypoint.sh

```diff
@@ -5,9 +5,9 @@ if [ $2 ] ; then
   export RAILWAY_TOKEN=$2
 fi
 
-railway up --service $1 || error_code=$?
+railway up --detach --service $1 || error_code=$?
```

Comment:
I think using detach won't wait until the deployment is finished. How about giving the option to use detach or not?